src/main/java/com/moneymoover/validator/Validator.java

```diff
@@ -53,4 +57,29 @@ public BigDecimal validateAmount (String amount) throws InvalidBigDecimalExcepti
 
         return value;
     }
+
+    /**
+     * This method is used to check if the converted value in a show-rate or exchange command
+     * is less than 0.01. If it is, it will truncate to 0 in the transaction history, resulting
+     * in a loss of money for the user. The method will print what the minimum amount that must
+     * be converted from the initial currency is.
+     *
+     * @param amount the converted value provided by the user
+     * @param inst the currency relationship used
+     * @return if the amount is valid
+     */
+    public boolean validateTargetValue (BigDecimal amount, Forex inst) {
+        if (amount.compareTo(new BigDecimal(0.01)) < 0) {
+            Ui ui = new Ui();
+            Currency init = inst.getInitial();
+            Currency targ = inst.getTarget();
+            Forex reverse = new Forex(targ, init);
+            BigDecimal allowedMin = reverse.convert(new BigDecimal(0.01));
+            allowedMin = allowedMin.setScale(2, RoundingMode.HALF_UP);
```

Comment:
As mentioned on telegram, changing this to RoundingMode.UP might solve the rounding issues.